server/api/expense-pool/index.js

```diff
@@ -8,6 +8,8 @@ router.use('/:id/expenses', expenseRouter);
 
 // POST => add a new pool
 router.post('/', controller.addPool);
+// GET => show all pools
+router.get('/', controller.getPools);
```

Comment:
I added it so I could see if all of my test entries were going through. Renaming to '/test'